lib/client/ippool.go

```diff
@@ -132,70 +133,21 @@ func (h *ipPools) convertMetadataToListInterface(m unversioned.ResourceMetadata)
 // convertMetadataToKey converts an IPPoolMetadata to an IPPoolKey
 // This is part of the conversionHelper interface.
 func (h *ipPools) convertMetadataToKey(m unversioned.ResourceMetadata) (model.Key, error) {
```

Comment:
i believe historically it was called a "helper" (not a fan of that name), and they's why the `h` has been carried with it